dbt_project.yml

```diff
@@ -0,0 +1,46 @@
+
+# Name your project! Project names should contain only lowercase characters
+# and underscores. A good package name should reflect your organization's
+# name or the intended use of these models
+name: 'databricks_billing'
+version: '1.0.0'
+config-version: 2
+
+# This setting configures which "profile" dbt uses for this project.
+profile: 'databricks_billing'
+
+# These configurations specify where dbt should look for different types of files.
+# The `model-paths` config, for example, states that models in this project can be
+# found in the "models/" directory. You probably won't need to change these!
+model-paths: ["models"]
+analysis-paths: ["analyses"]
+test-paths: ["tests"]
+seed-paths: ["seeds"]
+macro-paths: ["macros"]
+snapshot-paths: ["snapshots"]
+
+target-path: "target"  # directory which will store compiled SQL files
+clean-targets:         # directories to be removed by `dbt clean`
+  - "target"
+  - "dbt_packages"
+
+
+# Configuring models
+# Full documentation: https://docs.getdbt.com/docs/configuring-models
+
+# In this example config, we tell dbt to build all models in the example/ directory
+# as tables. These settings can be overridden in the individual model files
+# using the `{{ config(...) }}` macro.
+models:
+  databricks_billing:
+    # Config indicated by + and applies to all files under models/example/
+      marts:
+        materialized: view
+      staging:
+        materialized: view
+
+vars:
+  databricks_billing_database: SANDBOX
+  databricks_billing_schema: DEV_SUELEN_RODRIGUES
+## precisa ter uma tabela chamada databricks_billing
```

Comment:
Can we use only one language for the documentation? Someplaces are in english and another’s are in portuguese